packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -60,7 +61,10 @@
     "friday": "Friday",
     "saturday": "Saturday",
     "participantJoined": "joined the chat.",
-    "participantLeft": "left the chat."
+    "participantLeft": "left the chat.",
+    "editMessage": "Edit",
+    "removeMessage": "removeMessage",
```

Comment:
should the right hand side here be camel case or something like `remove message` or `delete message`?

Reply:
What... I think that was a copy&paste problem, thanks for catching it!